pipeline.yml

```diff
@@ -19,4 +18,9 @@ healthCheck: "/api/observations/about/health"
 # string parameter that tells the ALB how to route requests to the containers (e.g. /my-app/*); this forms a listener rule for the load balancer
 contextPath: "/api/observations/*"
 configRepoPath: "wma/docker/wdfn/time-series-services"
-configCredentialsId: "wma-eto-eb-rsa"
\ No newline at end of file
+configCredentialsId: "wma-eto-eb-rsa"
+urlRules:
+  - type: forward
+    conditions:
+      - "/api/observations/*"
+    priority: 302
```

Comment:
This should be `309` (i.e. the same value as the `rulePriority` parameter). Additionall, the `contextPath` and `rulePriorty` parameters don't do anything anymore and can be removed.